test/Microsoft.TestPlatform.Build.UnitTests/VsTestTaskTests.cs

```diff
@@ -234,5 +234,40 @@ public void CreateArgumentShouldAddOneCollectArgumentForEachCollect()
             Assert.IsNotNull(allArguments.FirstOrDefault(arg => arg.Contains("--collect:name1")));
             Assert.IsNotNull(allArguments.FirstOrDefault(arg => arg.Contains("--collect:\"name 2\"")));
         }
+
+        [TestMethod]
+        public void CreateArgumentShouldAddMultipleTestAdapterPaths()
+        {
+            var vstestTask = new VSTestTask();
+
+            // Add values for required properties.
+            vstestTask.TestFileFullPath = "abc";
+            vstestTask.VSTestFramework = "abc";
+
+            vstestTask.VSTestTestAdapterPath = new string[] { "path1", "path2" };
+
+            var allArguments = vstestTask.CreateArgument().ToArray();
+
+            Assert.IsNotNull(allArguments.FirstOrDefault(arg => arg.Contains("--testAdapterPath:path1")));
+            Assert.IsNotNull(allArguments.FirstOrDefault(arg => arg.Contains("--testAdapterPath:path1")));
+        }
+
+        [TestMethod]
+        public void CreateArgumentShouldAddMultipleLoggers()
+        {
+            var vstestTask = new VSTestTask();
+
+            // Add values for required properties.
+            vstestTask.TestFileFullPath = "abc";
+            vstestTask.VSTestFramework = "abc";
+
+            vstestTask.VSTestLogger = new string[] { "trx;LogFileName=foo bar.trx", "console" };
+
```

Comment:
Nit: extra line
Also, better for required properties, use new VSTestTask { TestFileFullPath = "abc", VSTestFramework = "xyz"}